src/pipelines.py

```diff
@@ -26,12 +28,12 @@ def retinanet(config, train_mode):
     if train_mode:
         return retinanet
 
-    postprocessor = postprocessing(retinanet, label_encoder, config)
+    postprocessor = postprocessing(retinanet, loader.get_step('label_encoder'), config)
```

Comment:
@apyskir I am crossed here since on one hand I do like the `.get_step` functionality but on the other label_encoder doesn't seem like a loader step.
What are your thoughts @taraspiotr @kamil-kaczmarek ?

Reply:
@jakubczakon I think it actually is a part of a loading process. A subsidiary step, but it is there.

Reply:
@apyskir I think it is more a part of a preprocessing process not loading itself